client/views/omnichannel/departments/DepartmentEdit.js

```diff
@@ -226,10 +230,24 @@ export function EditDepartment({ data, id, title, reload }) {
 
 	const formId = useUniqueId();
 
+	const tab = useRouteParameter('tab');
+	const hasCannedResponsesLicense = useHasLicense('canned-responses');
+	const cannedResponsesEnabled = useSetting('Canned_Responses_Enable');
+	const showCanned = hasCannedResponsesLicense && cannedResponsesEnabled && tab === 'canned-responses';
+
+	const handleOpenCannedResponses = useMutableCallback(() => {
+		router.push({ ...params, tab: 'canned-responses' });
+	});
+
+	const handleCloseCannedResponses = useMutableCallback(() => {
+		router.push({ ...params, tab: '' });
+	});
+
 	return <Page flexDirection='row'>
 		<Page>
 			<Page.Header title={title}>
 				<ButtonGroup>
+					{hasCannedResponsesLicense && cannedResponsesEnabled && <Button onClick={handleOpenCannedResponses} title={t('Canned Responses')}><Icon name='baloon-exclamation' size='x16'/></Button>}
```

Comment:
When the user clicks on the `Canned Response` button nothing happens.

Reply:
@renatobecker it's fixed now! I realized we have another `routes.js` inside `client/omnichannel`. It was apparently the problem and we have other duplicated routes there. IDK if there's any reason but if you need to let me know and I can help with some fix.

Reply:
Hey @dougfabris!!
Thanks for fixing the issue, but I noticed that the form is not validating either `shortcut` or `description, which is a problem since both are mandatory.

Reply:
@renatobecker already fixed. If I missed something, please let me know!